### PR TITLE
Updated the locale directory creation rule by using the iLib LocaleMatcher API

### PR DESCRIPTION
**Pull request by gouniLee**
* Updated the locale directory creation rule by using the iLib LocaleMatcher APIs
  * It will be consider to use `getLikelyLocaleMinimal()` when it's published
* Added test case to check resource path with webOS locale list (194)

### CODE REVIEW

File: JSONResourceFile.js [hunk 1]
```diff
@@ -36,13 +37,18 @@ var logger = log4js.getLogger("loctool.plugin.JSONResourceFile");
...
+    this.baseLocale = false;
 
     if (props) {
         this.project = props.project;
         this.locale = new Locale(props.locale);
         this.API = props.project.getAPI();
+
+        langDefaultLocale = new LocaleMatcher({locale: this.locale.language});
+        propsLocale = new LocaleMatcher({locale: this.locale});
+        this.baseLocale = ((langDefaultLocale.getLikelyLocale().getSpec() === propsLocale.getLikelyLocale().getSpec())? true:false);
```
Comment (ehoogerbeets) [inline JSONResourceFile.js]: Don't need the question-colon operator here:

```javascript
this.baseLocale = langDefaultLocale.getLikelyLocale().getSpec() === propsLocale.getLikelyLocale().getSpec();
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
 # ilib-loctool-webos-json-resource
-JSON resource file handler plugin for webos platform loctool
+ilib-loctool-webos-json-resource is a plugin for the loctool that
+allows it to read and localize JSON resource files. This plugins is optimized for webOS platform.
+
+## License
+
+Copyright © 2019, JEDLSoft
```
[the remark this answers is not in the document]
Comment (gouniLee) [reply README.md]: For now, I think it's ok to JEDLSoft as opensource work as a POC step. then when it comes to adopting actual webOS products, it's better to update LGE Inc.

File: test/testJSONResourceFilePaths.js [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+        test.expect(1);
+
+        var jsrf = new JSONResourceFile({
+            project: p
+        });
+        test.ok(jsrf);
+
+        test.done();
+    },    
+    teasJSONResourceFileGetResourceFilePaths: function(test) {
```
[the remark this answers is not in the document]
Comment (gouniLee) [reply test/testJSONResourceFilePaths.js]: oh, I updated it.

File: JSONResourceFile.js (hunk 1 above)
Comment (gouniLee) [reply JSONResourceFile.js]: I updated it

**Pull request merged**